manifest.webapp

```diff
@@ -66,5 +66,12 @@
       "type": ["io.cozy.contacts"],
       "href": "/services"
     }
-  ]
+  ],
+  "services": {
+    "checkAndUpdate": {
```

Comment:
a more explicit name can be great! This is about `creatingContactsAppIndex` or something similar, no?